app/presenters/activity_presenter.rb

```diff
@@ -1,30 +1,6 @@
-class ActivityPresenter < BasePresenter
+class ActivityPresenter < ActionPresenter
   presents :activity
-
-  def avatar_with_link(size)
-    h.link_to(avatar_image(size), 'javascript:void(0)')
-  end
-
-  def created_at_ago
-    h.local_time_ago(activity.created_at)
-  end
-
-  def icon
-    icon_css = %w{activity-icon fa}
-    icon_css << case activity.trackable_type
-                when 'Evidence'
-                  'fa-flag'
-                when 'Issue'
-                  'fa-bug'
-                when 'Node'
-                  'fa-folder-o'
-                when 'Note'
-                  'fa-file-text-o'
-                else
-                  ''
-                end
-    h.content_tag :span, nil, class: icon_css
-  end
+  collection :trackable
```

Comment:
`collection` may sound like a `has many` relationship?
Is there a particular reason to choose that name ?

Reply:
I just thought that `collection` best defines the polymorphic collection type name for the model of the presenter. Ideas?